src/_includes/checked-mode-2.0.html

```diff
@@ -0,0 +1,5 @@
+<aside class="alert alert-info" markdown="1">
+  **Dart 2.0 note**:
+  Checked mode will be obsolete in Dart 2.0 and will be removed. For more
```

Comment:
We could make this a little less repetitive. Maybe just:

Checked mode won't be in Dart 2.0. For more information...